hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiryManager.java

```diff
@@ -56,7 +56,11 @@ public void trackRecord(AccountID owner, long expiry) {
 		payerExpiries.track(owner.getAccountNum(), expiry);
 	}
 
-	public void resumeTrackingFrom(FCMap<MerkleEntityId, MerkleAccount> accounts) {
+	public void restartTrackingFrom(FCMap<MerkleEntityId, MerkleAccount> accounts) {
+		recordCache.reset();
+		txnHistories.clear();
+		payerExpiries.reset();
```

Comment:
The call to `payerExpiries.reset()` was not tested.

Reply:
Could not mock this entity. Added some more validations after restart tracking from accounts.